meta-mender-atmel/recipes-bsp/u-boot/u-boot-mender-atmel.inc

```diff
@@ -0,0 +1,14 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI_remove = " \
+	file://0005-fw_env_main.c-Fix-incorrect-size-for-malloc-ed-strin.patch \
+	file://0003-Integration-of-Mender-boot-code-into-U-Boot.patch \
+"
+
+SRC_URI_append = " \
+	file://0001-ARM-sama5d27_som1_ek-Add-support-to-mender.patch \
+	file://0002-Integration-of-Mender-boot-code-into-U-Boot.patch \
```

Comment:
Sure, I will do. In fact it's due to a more recent U-Boot (2018-07) on sumo (new release on meta-atmel).

Reply:
@mirzak should we add the newer u-boot patch to meta-mender and simply add it to SRC_URI here?  I'm sure there are other boards that will end up needing it.

Reply:
We should have this patch on master? Since the default on master is 2018.07 U-boot. Did you mean to back-port these to sumo as well?

@texierp , did you copy this patch from meta-mender/master?

Unrelated to this PR but we probably should keep all versions of the patches that we create following the version that is in poky/oe-core. Like we have done with [this one](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-core/recipes-bsp/u-boot/patches/0003-U-Boot-pre-2017.03-integration-of-Mender-boot-code-i.patch)

Reply:
Yeah, I guess if it's already there on master we should just see about backporting it to sumo rather than storing it in this repo.

Reply:
Thanks, sorry I did not take the time to look the patch on master (always on sumo). Indeed, should be better to backport this patch.